.github/ISSUE_TEMPLATE/flytectl_issue.yaml

```diff
@@ -0,0 +1,36 @@
+name: Flytectl Feature request
+description: Suggest a feature for flytectl
+title: "[Flytectl Feature]"
+labels: enhancement, untriaged, flytectl
+body:
+  - type: textarea
+    id: feature-description
+    validations:
+      required: true
+    attributes:
+      label: "Feature description"
+      description: "Describe the feature/command for flytectl"
+      placeholder: "A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] OR why this would be a cool addition OR which usecases could this unblock."
```

Comment:
```suggestion
      label: "Describe the feature/command for flytectl"
      description: "A clear and concise description of what the problem is."
      placeholder: "I'm frustrated when [...] OR this feature can be a cool addition because [...] OR this feature can simplify [...] use case."
```